spec/classes/foreman_proxy_content_spec.rb

```diff
@@ -25,6 +27,32 @@
             .with_rhsm_url("https://#{facts[:fqdn]}:443/rhsm")
         end
 
+        context 'with custom import/export paths' do
+          let(:params) do
+            # an array is not used for the export paths, to test that a Stdlib::Absolutepath value
+            # is properly handled as if it were a single element array
```

Comment:
This has the potential to get lost as it's not as "enforced" as a dedicated test case. I would rather lean towards just adding another context block that is dedicated to testing passing a string to both parameters, e.g.

```
context 'with custom import/export paths as strings' do

end
```

Reply:
OK, I've created separate test cases for array and string inputs.